index.js

```diff
@@ -87,7 +103,15 @@ function parseUnread() {
     if (err) {
       self.emit('error', err);
     } else if (results.length > 0) {
-      async.each(results, function( result, callback) {
+      
+      self.imap.setFlags(results, ['\\Seen'], function (err) {
+        if (err) {
+          console.log(JSON.stringify(err, null, 2));
+        }
+      });
```

Comment:
@dapanas is it necessary to set a message as \\Seen here? It voids markSeen options